plugin/generator/refreshable_type.go

```diff
@@ -174,6 +175,10 @@ func interfaceMethods(typ types.Type, typeSet RefreshableTypes) []jen.Code {
 		for i := 0; i < underlying.NumFields(); i++ {
 			field := underlying.Field(i)
 			if field.Exported() {
+				fieldOptions, _ := tags.ParseTag(underlying.Tag(i))
```

Comment:
Why are we ignoring the error in all cases? It may require more refactoring but if invalid tags are fatal then we need to bubble that up.

Reply:
I wanted to make it more obvious what I was doing first before refactoring and making this fatal. I can do that but it requires a lot of refactoring.